### PR TITLE
Split REVIEWs and match up with revised schedule

### PR DESCRIPTION
**Pull request by remarcmij**
- Added REVIEW topics for promises.
- Updated VSCode tips.

There still many review topics to be added.

### CODE REVIEW

File: Week7/REVIEW.md [hunk 1]
```diff
@@ -2,251 +2,11 @@
 
 ```
 This review covers:
- • Git Workflow
- • Map, 
- • Reduce 
- • Filter
+• Object Oriented Programming 
+• Code flow (order of execution) 
+• Async VS Sync
```
Comment (mkruijt) [inline Week7/REVIEW.md:7]: we moved async/sync to the next week and promises to this week

File: Week9/REVIEW.md [hunk 2]
```diff
@@ -1,262 +1,24 @@
...
-    return 'Hello ' + name;
-}
 ```
-
-Same as above with arrow (fat arrow) notation - shorthand notation. This is easy to mess up. Notice no return. 
-```js
-var sayHello2 = (name) => 'Hello ' + name;
+This review covers:
+• (re)writing data structures (in JSON)
+• Promises
```
Comment (mkruijt) [inline Week9/REVIEW.md:6]: see commend above promises is moved to week 7

**Review by mkruijt — CHANGES_REQUESTED** [review]
really nice @remarcmij I really like the different concepts in the fundamentals file. Maybe we can remove the weekly reviews entirely and just link to the specific files in the fundamentals? What do you think?

**Comment by remarcmij** [comment]
Hi @mkruijt, are you proposing to directly link the topics mentioned in the main README to the specific files in fundamentals? That sounds like a good idea. It removes one level of indirection and more urgently signals where we still have work to do. Is this something you would like to do or shall I make the changes?

**Comment by mkruijt** [comment]
yes that is what I'm suggesting. I can do it tomorrow 🙂 . If we merge this in the morning!

**Review by mkruijt — APPROVED** [review]

**Pull request merged**